src/EditorFeatures/CSharpTest/EditAndContinue/ActiveStatementTests.cs

```diff
@@ -8189,15 +8189,15 @@ static void F(object o)
         }
 
         [Fact]
-        public void MethodUpdate_VarDeconstruction()
+        public void MethodUpdate_VarDeconstructionInForLoop()
         {
             string src1 = @"
 class C
 {
     static void F(object o1, object o2)
     {
-        <AS:0>Console.WriteLine(1);</AS:0>
         for (var (x, y) = o1; ; ) { }
+        <AS:0>Console.WriteLine(1);</AS:0>
```

Comment:
That makes no difference to the outcome. I think I moved it to simplify my debugging (fewer steps to skip).